panelVisibilityManager.js

```diff
@@ -504,8 +509,13 @@ var PanelVisibilityManager = class HideTopBar_PanelVisibilityManager {
             ],
             [
                 this._settings,
-                'changed::enable-active-window',
-                this._updateIntellihideStatus.bind(this)
+                'changed::show-padding',
+                this._updateSearchEntryPaddingOrMargin.bind(this)
+            ],
+            [
+                this._settings,
+                'changed::hot-corner',
+                this._updateSettingsHotCorner.bind(this)
```

Comment:
This is probably a copy-paste mistake:
```suggestion
                'changed::show-padding',
                this._updateSearchEntryPaddingOrMargin.bind(this)
            ],
            [
                this._settings,
                'changed::enable-active-window',
                this._updateIntellihideStatus.bind(this)
```

Reply:
> Why would you make this change optional? Are there any drawbacks from using the padding instead of the margin property to define the search bar placement?

My logic here was I can only check on my manjaro gnome version. so not sure if there is another linux distro or version of gnome that may be better served keeping the margin instead of the padding.

Reply:
> This is probably a copy-paste mistake:

ah crud, I'll check it out.  good call out!